mpd/mpd.go

```diff
@@ -1099,6 +1100,21 @@ func (as *AdaptationSet) AddNewAccessibilityElement(scheme AccessibilityElementS
 	return accessibility, nil
 }
 
+// AddNewInbandEventStream - Adds a new InbandEventStream Descriptor to an adaptation set
+// uri - Scheme ID URI for the inband event stream
+// val - value for inband event stream
+func (as *AdaptationSet) AddNewInbandEventStream(uri *string, val *string) error {
```

Comment:
Let's be more inline with the rest of the functions in this file and make the parameters just be values `(uri string, val string)`, and set them as pointers when creating the `DescriptorType` e.g. `SchemeIDURI: Strptr(uri)`.

Reply:
Made the changes.